### ISSUE TITLE
Move language parameters from Pipeline to an operation parameters

### ISSUE THREAD

**Issue by dodijk** [textpipe/textpipe#39]
As discussed in https://github.com/textpipe/textpipe/pull/35#discussion_r201319799.

**Comment by anneschuth** [comment]
I'm starting to doubt this idea. It seems like language parameters are not parameters you'd want to pass when you create the pipeline, but rather parameters you'd want to pass when you *call* the pipeline with a document.

**Comment by VascoVisser** [comment]
Or language can be a document property. This allows smt like a language detection operation.

**Comment by anneschuth** [comment]
True, I think that's a good idea. But my point was that I don't think these language parameters are useful when creating a pipeline. So, in essence, I don't think this issue is an issue anymore. (Or more precise, I think it was never an issue in the first place).
